src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -204,16 +214,13 @@ public String getOgcCollectionFeatures(
 			params = collectionsParams.builder().collectionId(collectionId).bbox(bbox)
 				.paging(limit, startIndex, count)
 				.countries(countries).states(states).counties(counties)
-				.hydrologicUnits(hydrologicUnits).nationalAquiferCode(nationalAquiferCode)
-				.agencyCode(agencyCode).monitoringLocationNumber(monitoringLocationNumber)
+				.hydrologicUnits(hydrologicUnits).nationalAquiferCodes(nationalAquiferCodes)
+				.agencyCodes(agencyCodes).monitoringLocationNumbers(monitoringLocationNumbers)
 				.monitoringLocationType(monitoringLocationType)
 				.bbox(bbox). paging(limit, startIndex, count).build();
 
+			// This will not be null, even if the result set is empty, because metadata will be returned.
```

Comment:
meta data will only be returned if the collection id is valid. Otherwise, I believe null will be returned 
if the collection id was not found. Looks like maybe a missing test.

Reply:
I will take  look

Reply:
ah, got it. good catch.